src/components/LessonPlanList/LessonPlanRow/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+
+import { withStore } from '../../../Store';
+
+class LessonPlanRow extends Component
+{
+  onClick = event =>
+  {
+    const lessonPlan = this.props.lessonPlan;
+    const store = this.props.store;
+    store.updateCurrentLessonPlanID(lessonPlan.id);
+    delete lessonPlan.id;
+    store.updateCurrentLessonPlan(lessonPlan);
+  }
+
+  render()
+  {
+    const {name, date, description} = this.props.lessonPlan
+    return (
+      <div 
+        className="lesson_plan_row"
+        onClick={this.onClick}
+      >
+        <div className="name">
+        {name}
+        </div>
+        <div className="date">
+          {date.getFullYear()}/{date.getMonth()}/{date.getDay()}
```

Comment:
Don't quote me on this but I think that getMonth and getDay return zero-indexed values. I.e. `January => 0`

Reply:
You are correct about getMonth. getDate returns what I want (1 indexed) but getDay is wrong entirely.